### ISSUE TITLE
How to get fans to show up in HomeKit?

### ISSUE THREAD

**Issue by SawkeeReemo** [zyonse/homebridge-dreo#50]
**Describe Your Problem:**
How do you get the fans to show up in HomeKit? The only option in the config is to enter your Dreo account info. I’ve done that, added the child bridge to my HomeKit, but no fans have appeared anywhere. And when I go to add accessory, they are not listed. Both of my fan models are on your compatibility list.

Any ideas? I’m on the latest version of everything.

**Comment by zyonse** [comment]
Can you restart homebridge in debug mode and post the console output?

**Comment by SawkeeReemo** [comment]
I actually figured it out. Apparently when I signed up for my account with the DREO app, I used Bitwarden password vault to generate a password that was 23 characters long. DREO let me use it with no warning, and I was able to be logged in, add my fans, and control them. 

But I noticed that your plugin wasn’t able to retrieve my access token because login credentials were wrong, even after verifying that they were correct. 

I logged out of the DREO app, and generated a new password. This is when I noticed they have in fine print “password length 8-20 characters.” Even though they allowed me to have a password that was longer, I couldn’t log into the app with it, nor could your plugin. 

I changed my password to 20 characters, and now we’re good. This is a bug in the DREO app allowing an illegal password, not your plugin. 

Sorry for the false alarm, but hopefully my finding here can help someone else who might be looking.
